src/weather/app/README.md

```diff
@@ -0,0 +1,18 @@
+# WeatherApp
+
+Weather application.
+
+## How to Run
+
+Inside the folder, run:
+`$ iex -S mix`
+`$ App.Weather.temperature_of "Rio de Janeiro"`
+
+You can also run for multiple cities with:
+`$ iex -S mix`
+`$ cities = ["Rio de Janeiro", "Niteroi", "Sao Paulo", "Porto Alegre"]`
```

Comment:
Does it work for countries and neighbourhoods as well? Or only cities?

Reply:
It works for countries, cities, and neighborhoods.
I just added more examples to the documentation.